roqoqo-qryd/tests/integration/tweezer_devices.rs

```diff
@@ -669,7 +671,7 @@ fn test_phi_theta_relation() {
 // Test TweezerDevice two_tweezer_edges() method
 #[test]
 fn test_two_tweezer_edges() {
-    let mut device = TweezerDevice::new(None, None);
+    let mut device = TweezerDevice::new(None, None, None);
 
     assert_eq!(device.two_tweezer_edges().len(), 0);
 
```

Comment:
Maybe I just missed it, but I can't find the to/from QrydDevice tested here

Reply:
It's tested in `api_devices.rs`